libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -469,86 +587,101 @@ class CreatorNode {
           immediate
         }
       }
-      return axios(req)
+      return await axios(req)
```

Comment:
iuuc, this would maybe throw here versus before it would throw in the caller. is that right?